src/blocks/stars.css

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+
+  margin-right: 4px;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
```

Comment:
Avoid using tag names for styling. Instead of using ':first-child' and ':nth-child', consider using class names or other selectors that do not involve tag names.